src/rf/apps/workers/process.py

```diff
@@ -198,17 +202,48 @@ def emr_hand_off(self, record):
             layer_id=layer_id,
             status=enums.STATUS_THUMBNAILED)
         all_valid = len(layer_images) == len(valid_images)
-        some_images = len(layer_images) > 0
-        ready_to_process = some_images and all_valid
+        layer_has_images = len(layer_images) > 0
+        ready_to_process = layer_has_images and all_valid
 
         if ready_to_process:
             layer = Layer.objects.get(id=layer_id)
             status_updates.update_layer_status(layer.id,
                                                enums.STATUS_PROCESSING)
-            create_cluster(layer)
+            response = create_cluster(layer)
+            self.start_health_check(layer_id, response)
             return True
         else:
-            return some_images
+            return layer_has_images
+
+    def emr_heartbeat(self, record):
+        try:
+            data = record['data']
+            job_id = data['job_id']
```

Comment:
This name is accurate but might be a little confusing, considering that we pass a param called job-id to the chunk/mosaic steps. In that context, the value is the layer ID.